src/main/resources/static/js/subscription.js

```diff
@@ -371,7 +304,7 @@ jQuery(document).ready(function () {
             "searching": true,
             // Load data for the table's content from an Ajax source
             "ajax": {
-                "url": addBackendParameter(frontendServiceUrl + "/subscriptions"),
+                "url": addBackendParameter(getFrontEndServiceUrl() + "/subscriptions"),
```

Comment:
Is there any better way in JS to build the URL instead of "+"?

Reply:
Did not find any better way to build strings, There is String("h" + "e" + "j") but all it does is to ensure what is built is a String type.

Reply:
Maybe we should build our own "buildUrl" method that we can use then. What i'm worried about is all the trailing "/" that can make things complex. The "buildUrl" method should be able to build "www.something.com/" and "/foo/bar" to "www.something.com/foo/bar" and not "www.something.com//foo/bar".

Reply:
This can be achieved with a regex but perhaps make another story for this where every ajax url gets checked in a common url method.